### PR TITLE
Update README for beta1

### PR DESCRIPTION
**Pull request by flands**
Modifications for beta release:
* Add k8s example
* Fix broken links
* Make agent and collector configuration consistent
* Fix jaeger receiver syntax errors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,13 +60,23 @@ separately as either a Docker container, VM, or Kubernetes pod.
...
+Apply the [sample YAML](example/k8s.yaml) file:
 
-Create an Agent [configuration file](#configuration-file) as described above.
+```shell
+$ kubectl apply -f example/k8s.yaml
+```
+
+### <a name="getting-started-standalone"></a>Standalone
+
+Create an Agent [configuration](#config) file based on the options described below. By default, the agent has the opencensus receiver enabled, but no exporters.
```
Comment (pjanotti) [inline README.md]: Unfortunately, at this moment this is not true for the agent. It requires a config file and it needs the opencensus receiver explicitly there.

File: README.md [hunk 2]
```diff
@@ -131,21 +126,21 @@ receivers:
     address: "localhost:9411"
 
   jaeger:
-    collector_thrift_port: 14267
```
Comment (pjanotti) [inline README.md]: collector_* are the correct ones for Agent, the current differences between Collector and Agent are explained in receivers/README.md

File: README.md [hunk 3]
```diff
@@ -255,7 +245,7 @@ For example, to create a Docker image of the agent, tagged `v1.0.0`:
 
 and then the Docker image `v1.0.0` of the agent can be started  by
 ```shell
-docker run -v $(pwd)/config.yaml:/config.yaml  -p 55678:55678  ocagent:v1.0.0
+docker run --rm -it -v $(pwd)/ocagent-config.yaml:/conf/ocagent-config.yaml -p 55678:55678 -p 55679:55679 ocagent:v1.0.0
```
Comment (pjanotti) [inline README.md]: You need to pass `--config=/conf/ocagent-config.yaml` on the command-line

File: README.md [hunk 4]
```diff
@@ -308,7 +298,7 @@ $ ./bin/occollector_$($GOOS)
 (note: additional ports may be required depending on your receiver configuration):
 ```shell
 $ make docker-collector
-$ docker run --rm -it -p 55678:55678 occollector
+$ docker run --rm -it -v $(pwd)/occollector-config.yaml:/conf/occollector-config.yaml -p 55678:55678 -p 8888:8888 occollector
```
Comment (pjanotti) [inline README.md]: You need to pass `--config=/conf/occollector-config.yaml`

File: example/k8s.yaml [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+kind: ConfigMap
+metadata:
+  name: oc-agent-conf
+  labels:
+    app: opencensus
+    component: oc-agent-conf
+data:
+  oc-agent-config: |
+#    receivers:
+#      jaeger: {}
```
Comment (pjanotti) [inline example/k8s.yaml]: I'm not sure if the `{}` works for agent config.

File: README.md [hunk 6]
```diff
@@ -110,15 +105,15 @@ You should be able to see the traces in your exporter(s) of choice.
...
 
-The OpenCensus Service (both the Agent and Collector) is configured via a config.yaml file. In general, you need to
+The OpenCensus Service (both the Agent and Collector) is configured via a YAML file. In general, you need to
 configure one or more receivers as well as one or more exporters. In addition, diagnostics can also be configured.
 
 ### <a name="config-receivers"></a>Receivers
 
 A receiver is how you get data into the OpenCensus Service. One or more receivers can be configured. By default,
-the ``opencensus`` receiver is enabled on the OpenCensus Agent while no receivers are enabled on the OpenCensus Collector.
+the ``opencensus`` receiver is enabled on the OpenCensus Service (both the Agent and Collector).
```
[the remark this answers is not in the document]
Comment (flands) [reply README.md]: got it -- fixed

File: README.md (hunk 3 above)
Comment (flands) [reply README.md]: fixing

File: README.md (hunk 4 above)
Comment (flands) [reply README.md]: fixing

File: example/k8s.yaml (hunk 5 above)
Comment (flands) [reply example/k8s.yaml]: done

**Comment by flands** [comment]
@pjanotti incorporated feedback -- please take another look

**Pull request merged**